Actions/ReadSecrets/README.md

```diff
@@ -15,14 +15,17 @@ Additionally, the secrets specified by the authToken secret in AppDependencyProb
 | :-- | :-: | :-- | :-- |
 | shell | | The shell (powershell or pwsh) in which the PowerShell script in this action should run | powershell |
 | parentTelemetryScopeJson | | Specifies the parent telemetry scope for the telemetry signal | {} |
-| getSecrets | Yes | Comma separated list of secrets to get (add appDependencyProbingPathsSecrets to request secrets needed for resolving dependencies in AppDependencyProbingPaths) | |
+| getSecrets | Yes | Comma separated list of secrets to get (add appDependencyProbingPathsSecrets to request secrets needed for resolving dependencies in AppDependencyProbingPaths, add TokenForCommits in order to request a token to use for pull requests and commits) | |
```

Comment:
It looks like `TokenForCommits` will be either `GITHUB_TOKEN` or `GhTokenWorkflow` (depending on the value of `useGhTokenWorkflowForCommits`). Is this accurate?

Then why not follow the convention and use `getSecrets` as:
1. If `getSecrets` contains `GITHUB_TOKEN` use that for `TokenForCommits`.
2. If `getSecrets` contains `GhTokenWorkflow` use that for `TokenForCommits`.
3. If it contains both, use the weaker one. Or throw.

Reply:
Got it now, it's because `useGhTokenWorkflowForCommits` comes as a workflow input.